deploy/injector-mutating-webhook.yaml

```diff
@@ -22,10 +22,14 @@ webhooks:
         namespace: "vault"
       caBundle: ""
     rules:
+      - operations: ["CREATE"]
+        apiGroups: [""]
+        apiVersions: ["v1"]
+        resources: ["pods"]
       - operations: ["CREATE", "UPDATE"]
         apiGroups: [""]
         apiVersions: ["v1"]
-        resources: ["deployments", "jobs", "pods", "statefulsets"]
+        resources: ["deployments", "jobs", "statefulsets"]
```

Comment:
We can actually delete this whole entry as well - I'm not sure why we ever added this but it looks like a mistake to me. The first thing the handler does is try to unmarshal the object its given as a pod.